lib/CL/devices/cuda/pocl-ptx-gen.cc

```diff
@@ -0,0 +1,691 @@
+/* pocl-ptx-gen.cc - PTX code generation functions
```

Comment:
Will do - I ran `clang-format` but I forgot to check style of variable/function names etc.